src/main/java/com/timgroup/statsd/StatsDAggregator.java

```diff
@@ -0,0 +1,149 @@
+package com.timgroup.statsd;
+
+import com.timgroup.statsd.Message;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Queue;
+import java.util.Set;
+import java.util.Timer;
+import java.util.TimerTask;
+
+
+public class StatsDAggregator {
+    public static int DEFAULT_FLUSH_INTERVAL = 15000; // 15s
```

Comment:
15s is way to big, We're going to miss some dogstatsd bucket. We should find a sane default for all our client. What about 5s ? That way we push twice per dogstatsd bucket in the agent ?